.github/workflows/ci.yml

```diff
@@ -44,21 +63,19 @@ jobs:
 
       - name: Build wheels
         run: |
-          docker run -v $PWD:/root -w /root quay.io/pypa/manylinux1_x86_64 bash /root/bindings/python/tools/build_wheel.sh
```

Comment:
Can you remove the files `tools/build_wheel.sh` and `tools/build_wheel_macosx.sh` as part of this change?